frame/balances/src/lib.rs

```diff
@@ -891,11 +891,11 @@ where
 		_amount: T::Balance,
 		reasons: WithdrawReasons,
 		new_balance: T::Balance,
-	) -> Result {
+	) -> DispatchResult {
 		if reasons.intersects(WithdrawReason::Reserve | WithdrawReason::Transfer)
 			&& Self::vesting_balance(who) > new_balance
 		{
-			return Err("vesting balance too high to send value");
+			Err("vesting balance too high to send value")?
```

Comment:
personally I prefer `return Err("fooo".into())` because it more explicit than `Err("foo)?` but it's fine either way.